command/commands/user.py

```diff
@@ -97,23 +96,63 @@ def handle(self, command, user_id, channel):
             self.delete_helper(user_id, args.slack_id, channel)
 
         elif args.which == "edit":
-            # stub
-            msg = "user edited: "
-            if args.member is not None:
-                msg += "member: {}, ".format(args.member)
-            if args.name is not None:
-                msg += "name: {}, ".format(args.name)
-            if args.email is not None:
-                msg += "email: {}, ".format(args.email)
-            if args.pos is not None:
-                msg += "position: {}, ".format(args.pos)
-            if args.github is not None:
-                msg += "github: {}, ".format(args.github)
-            if args.major is not None:
-                msg += "major: {}, ".format(args.major)
-            if args.bio is not None:
-                msg += "bio: {}".format(args.bio)
-            return msg
+            param_list = {
```

Comment:
Was this picked up from someone else's commit? It doesn't seem relevant to this PR.